onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -472,7 +469,8 @@ void addOpSchemaSubmodule(py::module& m) {
   auto schemadef = m.def_submodule("schemadef");
   schemadef.doc() = "Schema submodule";
 
-  py::class_<ONNX_NAMESPACE::OpSchema> op_schema(schemadef, "OpSchema");
+  // Keep this binding local to this module
+  py::class_<ONNX_NAMESPACE::OpSchema> op_schema(schemadef, "OpSchema", py::module_local());
```

Comment:
Without these, the CI build fails as it seems these binding definitions clash with the ONNX Python bindings for the ONNX classes. hence, kept these bindings local to this module. (Not sure why this doesn't repro locally, but these bindings can be local to the onnxruntime module and that solves it)